46sshd/module-setup.sh

```diff
@@ -50,8 +50,9 @@ install() {
     /usr/bin/install -m 600 "$authorized_keys" \
             "$initdir/root/.ssh/authorized_keys"
 
-    inst_multiple /usr/sbin/sshd \
-        /etc/sysconfig/sshd
+    inst_simple /usr/sbin/sshd
+    inst_multiple -o /etc/sysconfig/ssh /etc/sysconfig/sshd
+
```

Comment:
I think that this should fail if neither is available, e.g.:

    if [ -f /etc/sysconfig/sshd ]; then
        inst_simple /etc/sysconfig/sshd
    else
        inst_simple /etc/sysconfig/ssh
    fi

Reply:
The lack of these files is perfectly legitimate, isn't it?

Reply:
You are right. For example on Fedora, the file just contains comments. It's also already marked optional in the service file.

Thus, this hunk is fine, as is.